message/tests.py

```diff
@@ -0,0 +1,53 @@
+from django.utils import timezone
+from .models import Message
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+import pytest
+
+AUTHOR = 'testuser@gmail.com'
+DOG_OWNER_ID = '123456789'
+DAYCARE_ID = '987654321'
+TEXT = 'Hello this is the test message'
+DATE = timezone.now()
+
+
+@pytest.fixture
+def create_message():
+    return Message.create(author=AUTHOR,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text=TEXT)
+
+
+@pytest.fixture
+def create_message2():
+    return Message.create(author=AUTHOR,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='This is reply to first message')
+
+
+@pytest.mark.django_db()
+class TestMessageModel:
+    def test_create_message(self, create_message):
+        assert create_message in Message.objects.all()
+
+    def test_delete_message(self, create_message):
+        assert create_message in Message.objects.all()
+
+        create_message.delete()
+        assert create_message not in Message.objects.all()
+
+    def test_get_all_messages_by_dog_owner_id(self, create_message):
+        dog_owner_messages = Message.get_all_dog_owner_message(create_message.dogowner_id)
+        assert create_message in dog_owner_messages
+
```

Comment:
Consider to pass to the method crease_message2 as you did in `test_get_all_messages_by_daycare_id` so the tests will be identifiers

Reply:
> Consider to pass to the method crease_message2 as you did in `test_get_all_messages_by_daycare_id` so the tests will be identifiers

added to test